react/admin/components/ReplyModal.tsx

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-console */
+import React, { FC, useState, useEffect } from 'react'
+import { defineMessages, WrappedComponentProps, injectIntl } from 'react-intl'
+import { Modal, Textarea, Button, Divider } from 'vtex.styleguide'
+import ReviewCellRenderer from './util/reviewCellRenderer'
+import ProductCellRenderer from './util/productCellRenderer'
+import { getUserInfo } from '../utils'
+
+const messages = defineMessages({
+  label: {
+    id: 'admin/reviews.reply.label',
+    defaultMessage: 'Reply',
+  },
+  button: {
+    id: 'admin/reviews.reply.button',
+    defaultMessage: 'Send',
+  },
+  info: {
+    id: 'admin/reviews.reply.info',
+    defaultMessage: 'This reply will be visible publicly',
+  },
+})
+const sessionPromise = getUserInfo()
+const useSessionResponse = () => {
+  const [session, setSession] = useState()
+  useEffect(() => {
+    if (!sessionPromise) {
+      return
+    }
+    sessionPromise.then(sessionResponse => {
+      const response: any = sessionResponse
+      setSession(response)
+    })
+  }, [])
+
+  return session
+}
+
+let isOpen = false
+
+const ReplyModal: FC<WrappedComponentProps> = ({
+  activeReview,
+  onReply,
+  intl,
+}) => {
+  const [state, setState] = useState({
+    reply: {
+      message: '',
+      adminUserId: '',
+    },
+  })
+
+  const sessionResponse: any = useSessionResponse()
```

Comment:
I think the backend can read the session cookies and get the values there instead of doing this on the frontend. What do you think?

Reply:
Indeed, will talk to Brian